docs/source/firmware.rst

```diff
@@ -31,11 +31,9 @@ Firmware
 
 This customized firmware that includes the modules listed in this documentation also includes a handful of features meant to make use
 of the available RAM on the BBC micro:bit in a more efficient manner. 
+The downside to this, is that we had to remove the built-in support for the micro:bit's compass and the SPI libraries, which are least likely to be needed.
```

Comment:
Can you make this stand out?  With a warning, or something like that? We don't want users to miss this little bit of info.